docs/technical_reference/property_models/modified_ADM1.rst

```diff
@@ -0,0 +1,345 @@
+Modified ADM1 Property Package
+==============================
+.. raw:: html
+
+    <style> .red {color:red} </style>
+    <style> .green {color:green} </style>
+    <style> .blue {color:blue} </style>
+
+.. role:: red
+
+.. role:: green
+
+.. role:: blue
+
+This package is an extension of the base Anaerobic Digestion Model no.1 (ADM1) and implements properties and reactions of an anaerobic digestion model for wastewater treatment using an anaerobic digester as provided in
```

Comment:
I would add a link to the original ADM1 model documentation here, referring the user to the base model.